src/Criterion.h

```diff
@@ -37,6 +37,21 @@ class Criterion : public AtomicMCDAObject  {
     ~Criterion();
     friend std::ostream & operator <<( std::ostream & out, const Criterion & crit );
 
+    /**
+     * generateDirection generator of random direction
+     * can also be used as reinitialisation for direction
+     * 
```

Comment:
nit: for the doc I think it's better if add the params in its description (with @param)

Reply:
corrected